tools/dockerfile/grpc_scan_build/Dockerfile

```diff
@@ -41,6 +41,19 @@ RUN git clone --recursive https://github.com/grpc/grpc.git
 EXPOSE 8181
 
 CMD \
+  # creating a repository for gRPC submodules for local cloning
+  # this list should be updated when the list of submodules for gRPC changes
+  mkdir grpc-submodules && cd grpc-submodules && \
```

Comment:
This would only populate the reference repo for grpc_scan_build Dockerfile, but we have 50 other docker images. What's up with the reference repo of all the other Dockerfiles.